examples/euler_gravity_3d/Makefile

```diff
@@ -0,0 +1,16 @@
+
+F2PY = f2py
+RIEMANN_SOURCE = ../../../riemann/src/
+
+all:
+	make euler_3d_gmap.so
```

Comment:
It is better to use the $(MAKE) variable then explicitly using `make` due to a variety of reasons.